api/clever/__init__.py

```diff
@@ -76,8 +77,7 @@ def external_type_from_clever_grade(grade):
     return CLEVER_GRADE_TO_EXTERNAL_TYPE_MAP.get(grade, None)
 
 
-class CleverAuthenticationAPI(OAuthAuthenticationProvider):
-
+class CleverAuthenticationAPI(OAuthAuthenticationProvider, ABC):
```

Comment:
I think this class is meant to be instantiated directly, since its included in the list here: 
https://github.com/ThePalaceProject/circulation/blob/e8d2e96c28f3cf7caad14f9aa73bfb42cacf2b1e/api/admin/controller/patron_auth_services.py#L30-L40

So I don't think we should be giving it the `ABC` type.

Reply:
Erk, yes. Not sure why that one was marked ABC...